2-repositories/scala-repo-server/src/main/scala/com.snowplowanalytics.iglu.server/util/ServerConfig.scala

```diff
@@ -28,16 +31,18 @@ object ServerConfig {
 
   val config = ConfigFactory.load
 
+  val env = System.getenv.asScala.lift
+
   //Interface on which the server will be running
   val interface = config.getString("repo-server.interface")
   //Port on which the server will be running
-  val port = config.getInt("repo-server.port")
+  val port = env("PORT").map(_.toInt) getOrElse config.getInt("repo-server.port")
 
-  val pgHost = config.getString("postgres.host")
-  val pgPort = config.getInt("postgres.port")
-  val pgDbName = config.getString("postgres.dbname")
-  val pgUsername = config.getString("postgres.username")
-  val pgPassword = config.getString("postgres.password")
+  val pgHost = env("RDS_HOSTNAME") getOrElse config.getString("postgres.host")
```

Comment:
I think `PG_` would be a bit more futureproof (for adding `ZK_HOSTNAME` or similar in the future).

Also, is it bad form for us not to be "vendoring" our environment variables - e.g. `IGLU_`?